spacecmd/src/spacecmd/shell.py

```diff
@@ -119,10 +119,9 @@ def __init__(self, options, conf_dir, config_parser):
                                     self.history_file)
                 except IOError:
                     logging.error(_N('Could not read history file'))
-        # pylint: disable=W0702
+        # pylint: disable=W0703
```

Comment:
what is W0703? Please use the symbolic name instead of the id. With just the id, everyone needs to look it up to understand what is being disabled.